src/shared/components/cancerSummary/CancerSummaryContent.tsx

```diff
@@ -476,7 +476,7 @@ export class CancerSummaryContent extends React.Component<ICancerSummaryContentP
                 {controls}
                 <SummaryBarGraph data={this.chartData} yAxis={this.yAxis} xAxis={this.xAxis} gene={this.props.gene} width={this.props.width}
                                  setPdfProperties={this.setPdfProperties} setPngAnchor={this.setPngAnchor} legend={this.showGenomicAlt}
-                                 orderedLabels={orderedLabels}
+                                 orderedLabels={orderedLabels} altCasesMax={this.altCasesMax}
```

Comment:
Seems like it would be better to derive this from the chartData passed in, but I guess we're already calculating this outside?

Reply:
In any case, we should bother because we're probably going to use a new library and hopefully that library will not try to pad a percentage axis above 100%